pkg/manifests/manifests_test.go

```diff
@@ -56,6 +63,30 @@ func TestManifests(t *testing.T) {
 		t.Errorf("invalid RouterClusterRoleBinding: %v", err)
 	}
 
+	if _, err := f.MetricsClusterRole(); err != nil {
+		t.Errorf("invalid MetricsRole: %v", err)
+	}
+
+	if _, err := f.MetricsClusterRoleBinding(); err != nil {
+		t.Errorf("invalid MetricsRoleBinding: %v", err)
+	}
```

Comment:
Copy-and-paste error: The method is named `MetricsClusterRoleBinding`, and the error message says "MetricsRoleBinding".